### PR TITLE
Allow menus collapsed by default

### PR DESCRIPTION
**Pull request by benoror**
Via `config.defaultMenuCollapsed`

Fix https://github.com/shuding/nextra/issues/109
Fix https://github.com/vercel/swr-site/issues/81

### CODE REVIEW

File: packages/nextra-theme-docs/src/misc/default.config.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export default {
   prevLinks: true,
   search: true,
   darkMode: true,
+  collapsed: false,
```
Comment (shuding) [inline packages/nextra-theme-docs/src/misc/default.config.js:13]: Thanks for contributing! I personally prefer a more explicit name, such as `defaultMenuCollapsed: false`, what do you think?

Comment (benoror) [reply packages/nextra-theme-docs/src/misc/default.config.js:13]: absolutely, good call!

**Review by shuding — APPROVED** [review]
Thank you!

**Comment by benoror** [comment]
Any chance to release this change, or waiting to get more delta?

**Pull request merged**